front/src/components/Family.jsx

```diff
@@ -65,7 +67,7 @@ class Family extends React.Component {
       method: 'post',
       headers: {
         'Content-Type': 'application/json',
-        'Authorization': this.props.token
+        'Autorization': this.props.token
```

Comment:
are you sure?

https://developer.mozilla.org/fr/docs/Web/HTTP/Headers/Authorization

Reply:
Modified Authorization and not Autorization...